.changeset/clean-seals-behave.md

```diff
@@ -0,0 +1,5 @@
+---
+"@jspsych-contrib/plugin-survey-slider": patch
```

Comment:
this should probably be major instead of patch, unless you don't want this to be marked as version 1.0.0

Reply:
Okay then sure, pushed that now